### PR TITLE
Add updates around third party library updates.

### LINKED ISSUE #8: Guard against third party unannotated methods
Third party libraries that have yet to annotate their headers for nullability will be imported into swift with IUO.

We should treat these as optionals and guard as we would any other optionals.

Example. A third party library that gets imported might look to swift as:

``` swift
public class ThirdPartyDelegate : NSObject {
    public func onError(errorInfo: ErrorInfo!)
}
```

An implementing subclass should:

``` swift
public class MyDelegate: ThirdPartyDelegate {
    override public func onError(errorInfo: ErrorInfo?) {
        logger.error("[ERROR] ThirdPartyDelegate: \(errorInfo?.message)")
    }
}
```

Specifically, note changing the type of ErrorInfo from `ErrorInfo!` to `ErrorInfo?`

### PR DESCRIPTION
**Pull request by eriklamanna**
Fixes #8

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -35,9 +35,10 @@
 * [Working with Storyboards](#storyboards)
 * [Functions vs Methods](#functions-vs-methods)
 * [Memory Management](#memory-management)
-  * [Extending Lifetime](#extending-lifetime)
+  * [Extending Lifetime](#extending-lifetime)Â
```
Comment (mattyohe) [inline README.markdown]: ~~Â~~

File: README.markdown [hunk 2]
```diff
@@ -833,6 +834,29 @@ while i < attendeeList.count {
     i += 1
 }
 ```
+
+## Third Party Imports
+
+Third party libraries that have yet to annotate their Objective-C headers for nullability will be imported into swift with implicitly unwrapped optionals. These should be treated as optionals and guarded on use as any other optional parameter. For example, a third party library that gets imported might look to swift as:
```
Comment (mattyohe) [inline README.markdown]: ~~swift~~ Swift

Comment (mattyohe) [inline README.markdown]: > a third party library that gets imported might look to swift as:

an Objective-C class imported into Swift might look like:

File: README.markdown [hunk 3]
```diff
@@ -833,6 +834,29 @@ while i < attendeeList.count {
...
+
+```swift
+public class ThirdPartyDelegate : NSObject {
```
Comment (mattyohe) [inline README.markdown]: Is `public` correct anymore? Would it be `open`? (Do we care?)

File: README.markdown [hunk 4]
```diff
@@ -833,6 +834,29 @@ while i < attendeeList.count {
...
+    public func onError(errorInfo: ErrorInfo!)
+}
+```
+
+An implementing subclass should implement this delegate by changing the implicitly unwrapped optional into an optional.
```
Comment (mattyohe) [inline README.markdown]: > An implementing subclass should implement this delegate by changing the implicitly unwrapped optional into an optional.

A overriding subclass should instead treat implicitly unwrapped optional parameters as optionals.

File: README.markdown [hunk 5]
```diff
@@ -833,6 +834,29 @@ while i < attendeeList.count {
...
+
+```swift
+public class MyDelegate: ThirdPartyDelegate {
+    override public func onError(errorInfo: ErrorInfo?) {
+        logger.error("[ERROR] ThirdPartyDelegate: \(errorInfo?.message)")
+    }
+}
+```
+
+Specifically, note changing the type of ErrorInfo from ErrorInfo! to ErrorInfo?
```
Comment (mattyohe) [inline README.markdown]: Specifically, the imported type `ErrorInfo!` has been changed to `ErrorInfo?`.

File: README.markdown (hunk 3 above)
Comment (eriklamanna) [reply README.markdown]: open works for me

**Pull request merged**